kaolin/conversions/meshconversions.py

```diff
@@ -78,7 +79,7 @@ def trianglemesh_to_voxelgrid(mesh: kaolin.rep.Mesh, resolution: int,
     if normalize:
         verts_max = mesh.vertices.max()
         verts_min = mesh.vertices.min()
-        mesh.vertices = (mesh.vertices - verts_min) / (verts_max - verts_min)
+        mesh.vertices = (mesh.vertices - verts_min) / (verts_max - verts_min) - 0.5
```

Comment:
Can you explain this modification ?

Reply:
The expected behaviour of the "normalize" argument is to center the vertices around the origin. This is now also clarified in the updated docstring. This restores previous behaviour expected by the examples.